src/cloud-cli/deploy.ts

```diff
@@ -39,7 +41,10 @@ export async function deploy(appName: string, host: string) {
     console.log(`Successfully deployed: ${appName}`);
     console.log(`${appName} ID: ${uuid}`);
   } catch (e) {
-    console.log(`Deploying ${appName} failed`);
-    throw e;
+    if (axios.isAxiosError(e)) {
+      console.error(`failed to deploy application ${appName}: ${e.response?.data}`);
```

Comment:
Actually, we can use our logger 
```ts
const logger = createGlobalLogger();
```

Reply:
Good idea. Done!

![Screenshot 2023-10-20 at 17 10 25](https://github.com/dbos-inc/operon/assets/3437048/345a7db3-3144-4fc4-bf34-00f7fceb8113)